docs/guides/mysql/backup/kubestash/auto-backup/index.md

```diff
@@ -0,0 +1,707 @@
+        storageRef:
+          namespace: demo
+          name: gcs-storage
+        retentionPolicy:
+          name: demo-retention
+          namespace: demo
+    # ============== Blueprint for Sessions of BackupConfiguration  =================
+    sessions:
+      - name: frequent-backup
+        sessionHistoryLimit: 3
+        scheduler:
+          schedule: "*/5 * * * *"
+        repositories:
+          - name: default-blueprint
+            backend: gcs-backend
+            directory: /default-blueprint
+            encryptionSecret:
+              name: encrypt-secret
+              namespace: demo
+        addon:
+          name: mysql-addon
+          tasks:
+            - name: logical-backup
+```
+
+Here,
+
+- `.spec.backupConfigurationTemplate.backends[*].storageRef` refers our earlier created `gcs-storage` backupStorage.
+- `.spec.backupConfigurationTemplate.sessions[*].schedule` specifies that we want to backup the database at `5 minutes` interval.
+
+Let's create the `BackupBlueprint` we have shown above,
+
+```bash
+$ kubectl apply -f https://github.com/kubedb/docs/raw/{{< param "info.version" >}}/docs/guides/mysql/backup/kubestash/auto-backup/examples/default-backupblueprint.yaml
+backupblueprint.core.kubestash.com/mysql-default-backup-blueprint created
+```
+
+Now, we are ready to backup our `MySQL` databases using few annotations. You can check available auto-backup annotations for a databases from [here](https://kubestash.com/docs/latest/concepts/crds/backupblueprint/).
+
+**Create Database**
+
+Now, we are going to create an `MySQL` CR in demo namespace. Below is the YAML of the MySQL object that we are going to create,
+
+```yaml
+apiVersion: kubedb.com/v1alpha2
+kind: MySQL
+metadata:
+  name: sample-mysql
+  namespace: demo
+  annotations:
+    blueprint.kubestash.com/name: mysql-default-backup-blueprint
+    blueprint.kubestash.com/namespace: demo
+spec:
+  version: "8.2.0"
+  replicas: 1
+  storageType: Durable
+  storage:
+    storageClassName: "standard"
+    accessModes:
+      - ReadWriteOnce
+    resources:
+      requests:
+        storage: 50Mi
+  terminationPolicy: WipeOut
+```
+
+Here,
+
+- `.spec.annotations.blueprint.kubestash.com/name: mysql-default-backup-blueprint` specifies the name of the `BackupBlueprint` that will use in backup.
+- `.spec.annotations.blueprint.kubestash.com/namespace: demo` specifies the name of the `namespace` where the `BackupBlueprint` resides.
+
+Let's create the `MySQL` we have shown above,
+
+```bash
+$ kubectl apply -f https://github.com/kubedb/docs/raw/{{< param "info.version" >}}/docs/guides/mysql/backup/kubestash/auto-backup/examples/sample-mysql.yaml
+mysql.kubedb.com/sample-mysql created
+```
+
+**Verify BackupConfiguration**
+
+If everything goes well, KubeStash should create a `BackupConfiguration` for our MySQL in demo namespace and the phase of that `BackupConfiguration` should be `Ready`. Verify the `BackupConfiguration` object by the following command,
```

Comment:
If everything is set up correctly, KubeStash will create a `BackupConfiguration` for our MySQL instance in the `demo` namespace. The phase of this `BackupConfiguration` should be `Ready`. You can verify the `BackupConfiguration` object by running the following command,